content/rc/administration/setup/customize-flexible-plan.md

```diff
@@ -0,0 +1,105 @@
+---
+Title: Customize a Flexible plan
+description:
+weight: 30
+alwaysopen: false
+categories: ["RC"]
+aliases: /rc/administration/customize-pro/
+---
+
+When you create a subscription with a Flexible plan, you customize it to your business needs.
+
+This means you:
+
+- Define deployment settings for your subscription, such as cloud provider, network settings, and so on
+
+- Create at least one database in order to determine cluster size.
```

Comment:
Maybe: "to determine the size of the underlying cluster."

Reply:
Fixed